keras_nlp/models/backbone.py

```diff
@@ -18,12 +18,10 @@
 
 from tensorflow import keras
 
-from keras_nlp.api_export import keras_nlp_export
 from keras_nlp.utils.python_utils import classproperty
 from keras_nlp.utils.python_utils import format_docstring
 
 
-@keras_nlp_export("keras_nlp.models.Backbone")
```

Comment:
Shouldn't we replace this with 

```
@keras.utils.register_keras_serializable(package="keras_nlp")
```

? And elsewhere?

Reply:
Good call.

So in total:
 - Sampler, Tokenizer: leave as is
 - Task, Backbone, Preprocessor: do not export, but do register as serializable 

Thanks!